### ISSUE TITLE
Viewed images do not load

### ISSUE THREAD

**Issue by delvh** [go-gitea/gitea#19651]
### Description

Since #19007, you can mark files in a PR as viewed.
Viewing an image, however, breaks viewing this image (see screenshots).
As soon as you `de-view` the image and reload the page, everything works as expected.

### Screenshots

# Normal/ Expected
![image](https://user-images.githubusercontent.com/51889757/167271775-69dc1400-cd24-4fc5-a89d-718733b330fb.png)

# Error Case
![image](https://user-images.githubusercontent.com/51889757/167271794-369a93df-a4d0-4370-8b1d-08db6bb5916e.png)

### Gitea Version

1.17.0+dev-546-g5ca224a78

### Can you reproduce the bug on the Gitea demo site?

Yes

### Operating System

Linux

### Browser Version

Firefox 99

**Comment by delvh** [comment]
I've gathered some additional information in the meantime, but I don't know how to fix it yet:
a) The screenshot above was for a SVG. "Luckily", this applies to images of all types, because I could confirm it with a PNG as well.
b) The reason for this issue is that both the enclosing span (and subsequently most likely through JS the image as well) get their width and height set to `0`:
![image](https://user-images.githubusercontent.com/51889757/172627579-f0c46dfa-089a-4b1e-bf4f-f3fef48ab815.png)
c) This leads me to assume that the width and height calculation fail because the image is not visible initially, and thus in good old JS fashion the calculated values default to 0 instead of throwing an error

I've looked through the corresponding code, but didn't find anything that looked as if it would support this hypothesis.
